internal/conv/conv.go

```diff
@@ -1,12 +1,27 @@
 package conv
 
+import "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
 func PString(s string) *string {
 	if s == "" {
 		return nil
 	}
 	return &s
 }
 
+func PSlices(s *schema.Set) *[]string {
```

Comment:
there's a function already called `schemaSetToSliceString` which you could use which converts a set to a `[]string`. You could use this func or move it to somewhere general to use it? 

https://github.com/resolutionlife/terraform-provider-looker/blob/main/looker/resource_looker_user_role.go#L209

Reply:
forgot about that one, I've moved it to the conv package along side a generic PSlices function

Reply:
though I'm still unsure if that's the best place for it, since everything else in that package is about returning pointer?

Reply:
Yep I'd agree - I'll ask around to see where is best to put this. Maybe in a helpers?

Reply:
maybe a util package or something hidden inside `internal`?